From 7df92e1360b35c5ba13f4581a4fd54dc26fd8f63 Mon Sep 17 00:00:00 2001
From: Julien Schueller <schueller@phimeca.com>
Date: Wed, 21 Jul 2021 20:03:20 +0200
Subject: [PATCH] Fix erroneous semicolon

---
 src/Interfaces/BonOsiTMINLPInterface.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/Interfaces/BonOsiTMINLPInterface.cpp b/src/Interfaces/BonOsiTMINLPInterface.cpp
index 09a101ef..0f7cf342 100644
--- a/Bonmin/src/Interfaces/BonOsiTMINLPInterface.cpp
+++ b/Bonmin/src/Interfaces/BonOsiTMINLPInterface.cpp
@@ -1853,7 +1853,7 @@ OsiTMINLPInterface::getConstraintsViolation(const double *x, double &obj)
       if(rowLower[i] > -1e10)
          rowViolation = std::max(0.,rowLower[i] - g[i]);
 
-      if(rowUpper[i] < 1e10);
+      if(rowUpper[i] < 1e10)
         rowViolation = std::max(rowViolation, g[i] - rowUpper[i]);
 
       norm = rowViolation > norm ? rowViolation : norm;
